### ISSUE TITLE
Multiple lines within @docs cause compile errors after formatting.

### ISSUE THREAD

**Issue by rupertlssmith** [avh4/elm-format#397]
Tried on versions 0.6.1-alpha and 0.7.0-exp so far.

If there are multiple lines following an @docs like this:

```
@docs my, awesome, elm,
  code, does, cool, stuff

```
If gets formatted like this:
```

@docs my, awesome, elm,
code, does, cool, stuff
```

The compiler then complains that the items 'code', ..., 'stuff' are not mentioned in the @docs.

The formatter should do one of the following.

* preserve the whitespace indents on multiple @docs lines so the compiler interprets them correctly.
* put everything on one line 
* insert @docs at the start of each line

**Comment by avh4** [comment]
Thanks for the report!  I have never seen code like that before, and didn't realize if was valid in elm-compiler.

**Comment by avh4** [comment]
Fixed in 9be89c82...5ba6041b.  Additional `@docs` will be added.
